pyprof/parse/kernel.py

```diff
@@ -121,7 +124,8 @@ def setRunTimeInfo(self, info):
         self.tid = info['tid']
         self.objId = info['objId']
         assert (self.rStartTime < self.rEndTime)
-        assert (self.rStartTime < self.kStartTime)
+        if self.kStartTime:
+            assert (self.rStartTime < self.kStartTime)
```

Comment:
While this is practically true, I vaguely remember having this assertion and it getting triggered. I did not investigate it but it could be due to very slight differences in the CPU and GPU timestamps (unix timestamps of whatever they use).

Reply:
I have not seen this be an issue - I can change it to self.rStartTime <= self.kStartTime.  

Or I can remove it - but it looks to me like these scripts are biased to using assertions as much as possible

Reply:
Yes, I have used as many assertions as possible. `self.rStartTime <= self.kStartTime` will have the same issue as I mentioned. We can move it to a separate PR and try it out on a variety of networks.